### ISSUE TITLE
New Terms: Death in mid, late adulthood

### ISSUE THREAD

**Issue by pdl** [obophenotype/human-phenotype-ontology#6840]
**Preferred term label:** Death in mid-adulthood
**Synonyms:** Death in middle age
**Definition (free text, please give PubMed ID): **Death between the age of 40 and 60 years. (c.f. https://hpo.jax.org/app/browse/term/HP:0003596)
**Parent term (use hpo.jax.org/app):** https://hpo.jax.org/app/browse/term/HP:0011420
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
**Your nano-attribution (ORCID):** Requested by Helen Firth ( https://orcid.org/0000-0002-6410-0882 )

**Preferred term label:** Death in late adulthood
**Synonyms:** -
**Definition (free text, please give PubMed ID):** Death at an age of at least 60 years.
**Parent term (use hpo.jax.org/app):** https://hpo.jax.org/app/browse/term/HP:0011420
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
**Your nano-attribution (ORCID):** Requested by Helen Firth ( https://orcid.org/0000-0002-6410-0882 )

We intend to store when patients have died in DECIPHER and express this in HPO, but we can only do the latter for ages up to 40 years.

**Comment by pdl** [comment]
Sorry, pressed some key combination which submitted too quickly. Editing now.

**Comment by pnrobinson** [comment]
thanks, we have added these terms.
